### ISSUE TITLE
FunctionName()

### ISSUE THREAD

**Issue by hossimo** [hossimo/GMA3Plugins#37]
Add Dump() Info page based on https://help2.malighting.com/Page/grandMA3/lua_object_dump/en/1.9/Dump

**Comment by Wile-E-Coyote-lab** [comment]
The page does not exist!

**Comment by hossimo** [comment]
yeah it's from 1.9, in the new version it's https://help.malighting.com/grandMA3/2.2/HTML/lua_object_dump.html Though I never ended up linking back to it.
